WMF Framework/Theme.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+
+@objc(WMFThemeColors)
+public class ThemeColors: NSObject {
+    fileprivate static let base10 = UIColor.wmf_color(withHex: 0x222222)!
```

Comment:
As an aside, it may be nice to add a comment with a link to the wiki page or ticket where these color names are defined.